data/event-schemas/examples/m.room.message$m.key.verification.request.yaml

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
this makes an example appear under the general definition of `m.room.message` (https://pr1271--matrix-spec-previews.netlify.app/client-server-api/#mroommessage), which feels confusing to me.

Worse, where there *ought* to be an example, is just an empty "Examples" heading:

![image](https://user-images.githubusercontent.com/1389908/195170459-521a01d3-0623-4f78-ade0-ed13f8d26e95.png)